src/ui/browser.ts

```diff
@@ -134,12 +135,21 @@ export class FileTableData extends AbstractTableData<File | null> {
           margin: 5px;
           fill: var(--icon-color, black);
         }
+
+        .${FileTableData.docIconClass} {
+          display: inline-block;
+          width: var(--doc-icon-width, 22px);
+          height: var(--doc-icon-height, 22px);
+          vertical-align: middle;
+          margin: 5px;
+          fill: var(--doc-icon-color, black);
+        }
         
         .${FileTableData.hoverImageClass} {
           position: absolute;
           top: 50%;
-          transform: translate(0%, -50%);
-          max-height: 90%;
+          transform: translate(0%, -70%);
+          height: 400px;
```

Comment:
This ensures on-hover images of thumbnails aren't too large. With the changes to icon sizes, hover images would otherwise be very large because max-height is set to 90%.